package.json

```diff
@@ -0,0 +1,88 @@
+{
+  "name": "rif-marketplace-meta",
+  "version": "1.0.0",
+  "description": "meta repository for convenient and fast local development of the rif marketplace",
+  "main": "index.js",
+  "scripts": {
+    "test": "echo \"Error: no test specified\" && exit 1",
+    "node:install": "npm i && meta npm install --include-only $npm_package_config_node_dirs --parallel",
+    "mkp:install": "npm i && meta npm install --include-only $npm_package_config_mkp_dirs --parallel",
+    "all:install": "npm run node:install",
+    "ganache-cli": "npx ganache-cli -d --db=./data/ganache/database --networkId=8545 --verbose=true --gasLimit=10000000 --allowUnlimitedContractSize",
+    "ganache-cli:mining": "npx ganache-cli -d --db=./data/ganache/database --networkId=8545 --verbose=true --gasLimit=10000000 --allowUnlimitedContractSize --blockTime=15",
+    "ganache-cli:pm2": "pm2 start npx --name=ganache -- ganache-cli -d --db=./data/ganache/database --networkId=8545 --verbose=true --gasLimit=10000000 --allowUnlimitedContractSize",
+    "contracts:deploy": "meta exec \"npx truffle migrate --network ganache\" --include-only $npm_package_config_contracts",
```

Comment:
sooo..... I take it one does not need to always deploy rns first anymore?

Reply:
not sure what you mean..